### DISCUSSION TITLE
textfield change label color from blue and when selected change label color

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#594] [Q&A]
![Screenshot 2022-11-19 10:11:40](https://user-images.githubusercontent.com/52072811/202831493-ae08fdb9-a47c-45e1-9434-47dd1db4c722.png)
I want the textfield when I don't click the label which was originally black. I want to change it to another color and when it's clicked, the blue color on the label changes to another color but it looks like the **label_color** option doesn't exist.  even though I want the default to not be black.

TextField(
		border="none",
		label="Search new Movie",
		color="white",
		border_color="white",
		bgcolor="#242444",
		focused_bgcolor="#242444",
		focused_color="white",
		cursor_color="white"
		)

I want the default label to be white and when I click on it, the label stays white

and one more .  how to use text_style I do not understand.  and hint_style .  the explanation is incomplete.  and there are no examples

**Comment by ndonkoHenri** [comment]
You have to use `label_style`.

https://flet.dev/docs/controls/textfield#label_style

I will search for an example, and if there isn't I will create one.

**Comment by ndonkoHenri** [comment]
There are no examples on this. So [i made one](https://github.com/flet-dev/examples/pull/44/commits/2829582fef87a5721a7be302439b1eb0b9e4cbdc) and made a [PR](https://github.com/flet-dev/examples/pull/44) to flet-dev/examples;
